Cogs/Helpers/verificationHelpers.py

```diff
@@ -17,6 +18,9 @@ def insert_verified_user_record(user_id, email, name):
     cursor.execute("INSERT INTO verified_users (user_id, email, time, full_name) VALUES (%s, %s, TIMESTAMP %s, %s);", [user_id, email, time, name])
     if(cursor.rowcount != 1):
         print(f"failed to insert verification record ({user_id}, {email}, {time})")
+    else:
+        await add_email(email)
```

Comment:
If im understanding this correctly, we automatically subscribe anyone who registers to the mailing list? We should probably have a command for them to unsubscribe too (unless thats linked within the email). Also should we write a script that autosubscribes everyone whos already in the db to the mailing list?

Reply:
I'm still working on the whole mailing list thing. It will mainly be through the website, but I want to give us more chances to reach out to people, so verification will subscribe them.

I will add dedicated subscribe and unsub commands too. I was just going to manually copy the emails from the verified records to the website db which'll have the list (partly why I wanted to combine the dbs, but its trivial to copy a bunch of emails).